cli/azd/pkg/project/service_target_containerapp.go

```diff
@@ -22,6 +30,11 @@ type containerAppTarget struct {
 	containerHelper     *ContainerHelper
 	containerAppService containerapps.ContainerAppService
 	resourceManager     ResourceManager
+	armDeployments      *azapi.StandardDeployments
+	console             input.Console
+	commandRunner       exec.CommandRunner
+
+	bicepCli func() (*bicep.Cli, error)
```

Comment:
I made this change intentionally as it didn't feel right to have `bicepCli` injected into the constructor, which leads to fetching of Bicep CLI **unconditionally** in container app services.